acceptance-tests/README.md

```diff
@@ -35,6 +35,19 @@ mvn clean install -DskipTests -DcleanNode
 This is mainly for CI servers. It starts the selenium docker container and runs all 
 nightwatch and java ATH tests in one shot.
 
+### Run tests against a local instance
+
+ATH looks for a properties file at `~/.blueocean-ath-config` (or any combination of similar command line options).
+These options will allow you to run a local ChromeDriver and execute the ATH against a local development instance
+that you can debug/etc.. Currently these properties are supported and/or necessary:
+```webDriverType=chrome
+webDriverUrl=http://localhost:9515
+webDriverBrowserSize=1024x768
+jenkinsUrl=http://localhost:8080/jenkins
+adminUsername=admin
+adminPassword=admin
```

Comment:
camel case is a bit odd for properties file but doesn't really matter.

Reply:
it's a java properties file, this is normal.

Reply:
Is there an external link we can provide to setting up a local ChromeDriver? I haven't done it myself but it'd be helpful for everyone to know.